baseapp/baseapp.go

```diff
@@ -341,9 +343,13 @@ func (app *BaseApp) MountStore(key storetypes.StoreKey, typ storetypes.StoreType
 	app.cms.MountStoreWithDB(key, typ, nil)
 }
 
-// LoadLatestVersion loads the latest application version. It will panic if
-// called more than once on a running BaseApp.
+// LoadLatestVersion loads the latest version from the commit multi-store. In
+// this context version == block height. It will panic if called more than once
+// on a running BaseApp.
+//
+// Side-effect: calls baseapp.Init()
 func (app *BaseApp) LoadLatestVersion() error {
+	app.logger.Debug(fmt.Sprintf("loading latest version %v\n", app.cms.LatestVersion()))
```

Comment:
depending on how many times we call this function, we might want to use the lazy version of fmt.sprintf since this is a debug statement and we usually won't actually be running it